api/src/main/java/jakarta/data/repository/Delete.java

```diff
@@ -25,27 +25,18 @@
 import java.lang.annotation.Target;
 
 /**
- * <p>Annotates a repository method to perform delete operations.</p>
+ * <p>Lifecycle annotation for repository methods which perform delete operations.</p>
  *
- * <p>The {@code Delete} annotation indicates that the annotated repository method requests one or more
- * entities to be removed from the database. To request deletion of specific entity instances, the annotated
- * repository method must have a single parameter whose type must be one of the following:
+ * <p>The {@code Delete} annotation indicates that the annotated repository method deletes the state of one or more
+ * entities from in the database. The annotated repository method usually has exactly one parameter whose type is
```

Comment:
My update to get rid of "usually" ended up disappearing. Here it is again,
```suggestion
 * entities from the database. To delete specific entity instances,
 * the annotated repository method must have exactly one parameter whose type is
```

Reply:
@njr-11 the trouble is I just don't know what this means. What is a "specific entity instance"?

IMO, the `delete()` method with no parameters deletes "specific entity instances". Specifically, it deletes all of them!

I don't think this phrase actually does the work you want it to do.

Reply:
Yeah, I think my previous language which I've forgotten now was better than this. I gave it another try.